### ISSUE TITLE
Idea - Change behaviour when player moves out of field

### ISSUE THREAD

**Issue by Cetty** [Joecorleone/Derailer#7]

**Comment by Cetty** [comment]
Different options are possible when player moves out of field:

- 

1. Player loses / gets destroyed
2. Player comes out of opposing connection
3. Player comes out at random connection

I think I would prefer option 2 or 3 as this increases playing time and I find it strange that someone losses when moving out of field.

**Comment by Joecorleone** [comment]
I agree, that it is not obvious, that moving out of field is a loss condition ...
However, I actually like that the challenge is to keep away from other players and the sides and redirect them into the field sides ... So, I would definitely like to have the option to choose
